### PR TITLE
Changelog for 1.10.4-stable and 1.11.1-beta

### PR DESCRIPTION
**Pull request by 5chdn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-This is the Parity 1.11.0-beta release! Hurray!
+This is the Parity 1.11.1-beta release! Hurray!
 
 Notable changes in reversed alphabetical order:
 
 - TOOLING: **Whisper CLI** [#8201](https://github.com/paritytech/parity/pull/8201)
   - `whisper-cli` is a standalone tool to communicate with the Whisper protocol.
   - It provides functionality to specify `whisper-pool-size`, `port` and `address` to use.
   - All whisper RPC APIs are enabled and can be directly acessed.
+I'm not used to writing these changelogs but I guess that would explain it
```
Comment (niklasad1) [inline CHANGELOG.md]: Remove :smiley:

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,6 +48,113 @@ Notable changes in reversed alphabetical order:
 
 The full list of included changes:
 
+- Backports ([#8624](https://github.com/paritytech/parity/pull/8624))
+  - Trace precompiled contracts when the transfer value is not zero ([#8486](https://github.com/paritytech/parity/pull/8486))
+    - Trace precompiled contracts when the transfer value is not zero
+    - Add tests for precompiled CALL tracing
+    - Use byzantium test machine for the new test
+    - Add notes in comments on why we don't trace all precompileds
```
Comment (niklasad1) [inline CHANGELOG.md]: I don't understand this sentence, can you rephrase? I guess it should be something like `we don't trace all precompiled contracts` ?!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
 
-This is the Parity 1.11.0-beta release! Hurray!
+This is the Parity 1.11.1-beta release! Hurray!
 
 Notable changes in reversed alphabetical order:
 
 - TOOLING: **Whisper CLI** [#8201](https://github.com/paritytech/parity/pull/8201)
   - `whisper-cli` is a standalone tool to communicate with the Whisper protocol.
   - It provides functionality to specify `whisper-pool-size`, `port` and `address` to use.
   - All whisper RPC APIs are enabled and can be directly acessed.
```
Comment (Tbaut) [inline CHANGELOG.md]: s/acessed/accessed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
   - This changes the behaviors of `eth_call` to respect VM errors if any.
   - In case of `REVERT`, it will also return the reverted return data in hex format.
-- ENGINES: **Block reward contract** [#8419](https://github.com/paritytech/parity/pull/8419)
+- ENGINES: **Block Reward Contract** [#8419](https://github.com/paritytech/parity/pull/8419)
   - The _AuRa_ PoA engine has now support for having a contract to calculate the block rewards.
   - The engine passes a list of benefactors and reward types to the contract which then returns a list of addresses and respective rewards.
-- CORE: **Private transactions integration pr** [#6422](https://github.com/paritytech/parity/pull/6422)
+- CORE: **Private Transactions** [#6422](https://github.com/paritytech/parity/pull/6422)
   - Parity now provides a private transactions system.
   - Please, check out our wiki to get and [overview and setup instructions](https://wiki.parity.io/Private-Transactions.html).
```
Comment (Tbaut) [inline CHANGELOG.md]: s/and/an

File: CHANGELOG.md (hunk 2 above)
Comment (Tbaut) [inline CHANGELOG.md]: s/precompileds/precompiled

File: CHANGELOG.md [hunk 5]
```diff
@@ -47,6 +48,113 @@ Notable changes in reversed alphabetical order:
 
 The full list of included changes:
 
+- Backports ([#8624](https://github.com/paritytech/parity/pull/8624))
+  - Trace precompiled contracts when the transfer value is not zero ([#8486](https://github.com/paritytech/parity/pull/8486))
+    - Trace precompiled contracts when the transfer value is not zero
```
Comment (Tbaut) [inline CHANGELOG.md]: This line is duplicated

File: CHANGELOG.md (hunk 1 above)
Comment (5chdn) [reply CHANGELOG.md]: jesus why do I keep copy-pasting this in xD

File: CHANGELOG.md (hunk 2 above)
Comment (5chdn) [reply CHANGELOG.md]: It's a commit message. I am still undecided if we should keep all of them in the changelogs.

File: CHANGELOG.md (hunk 5 above)
Comment (5chdn) [reply CHANGELOG.md]: Coincidence: Title of PR vs first commit message.
